lib/Elastica/Aggregation/DateHistogram.php

```diff
@@ -143,14 +143,16 @@ public function setFormat($format)
      */
     public function setExtendedBounds($min = '', $max = '')
     {
-        $bounds = array();
-        $bounds['min'] = $min;
-        $bounds['max'] = $max;
         // switch if min is higher then max
-        if (strtotime($min) > strtotime($max)) {
-            $bounds['min'] = $max;
-            $bounds['max'] = $min;
-        }
+        $bounds = strtotime($min) > strtotime($max)
```

Comment:
I'm aware that this could be a potential performance improvement, but as I don't see this as something that could have a real impact on the overall performance of an app code readability is more important for me.